main/src/main/java/org/mobilitydata/gtfsvalidator/notice/PlatformWithoutParentStationNotice.java

```diff
@@ -22,6 +22,8 @@
  * A platform has no `parent_station` field set.
  *
  * <p>This is different from {@code LocationWithoutParentStationNotice} since it is less severe.
+ *
+ * <p>Severity: {@code SeverityLevel.ERROR}
```

Comment:
This needs to be a WARNING. The spec says parent_station is optional for platforms.